src/commands/server/dashboard.ts

```diff
@@ -0,0 +1,52 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { Command, flags } from '@oclif/command'
+import { cli } from 'cli-ux'
+import * as notifier from 'node-notifier'
+
+import { CheHelper } from '../../api/che'
+import { cheNamespace } from '../../common-flags'
+
+export default class Dashboard extends Command {
+  static description = 'Open Eclipse Che dashboard'
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    chenamespace: cheNamespace,
+    url: flags.boolean({ description: 'Prints Eclipse Che dashboard url' })
+  }
+
+  async run() {
+    const { flags } = this.parse(Dashboard)
+
+    try {
+      const cheHelper = new CheHelper(flags)
+      const cheURL = await cheHelper.cheURL(flags.chenamespace)
+      const dashboardUrl = `${cheURL}/dashboard/`
+
+      if (flags.url) {
+        cli.url(dashboardUrl, dashboardUrl)
+      } else {
+        cli.info(`Opening ... ${dashboardUrl}`)
+        await cli.open(dashboardUrl)
+      }
+    } catch (error) {
+      this.error(error)
+    }
+
+    notifier.notify({
```

Comment:
won't it be executed even in case of error form catch block above?

Reply:
No, `this.error` prints the error and terminates the program.